thermal/thermal-daemon/thermal-conf.xml

```diff
@@ -1,6 +1,118 @@
 <?xml version="1.0"?>
 <!-- BEGIN -->
 <ThermalConfiguration>
+	<Platform>
+		<Name> TGL NUC i5  </Name>
+		<ProductName> NUC11PAHi5 </ProductName>
+		<Preference>QUIET</Preference>
+		<ThermalZones>
+			<ThermalZone>
+				<Type>cpu_passive</Type>
+				<TripPoints>
+					<TripPoint>
+						<SensorType>x86_pkg_temp</SensorType>
+						<Temperature>85000</Temperature>
+						<Type>Passive</Type>
+						<CoolingDevice>
+							<Type>rapl_limit_1</Type>
+						</CoolingDevice>
+					</TripPoint>
+					<TripPoint>
+						<SensorType>x86_pkg_temp</SensorType>
+						<Temperature>95000</Temperature>
+						<Type>Passive</Type>
+						<CoolingDevice>
+							<Type>rapl_limit_2</Type>
+						</CoolingDevice>
+					</TripPoint>
+				</TripPoints>
+			</ThermalZone>
+			<ThermalZone>
+				<Type>cpu_critical</Type>
+				<TripPoints>
+					<TripPoint>
+						<SensorType>x86_pkg_temp</SensorType>
+						<Temperature>99000</Temperature>
+						<Type>Critical</Type>
+					</TripPoint>
+				</TripPoints>
+			</ThermalZone>
+		</ThermalZones>
+		<CoolingDevices>
+			<CoolingDevice>
+				<Type>rapl_limit_1</Type>
+				<Path>/sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/constraint_0_power_limit_uw</Path>
+				<MinState>25000000</MinState>
+				<ReadBack>0</ReadBack>
+				<MaxState>13000000</MaxState>
+				<IncDecStep>-200000</IncDecStep>
+			</CoolingDevice>
+			<CoolingDevice>
+				<Type>rapl_limit_2</Type>
+				<Path>/sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/constraint_0_power_limit_uw</Path>
+				<MinState>13000000</MinState>
+				<ReadBack>0</ReadBack>
+				<MaxState>6000000</MaxState>
+				<IncDecStep>-200000</IncDecStep>
+			</CoolingDevice>
+		</CoolingDevices>
+	</Platform>
+	<Platform>
+		<Name> TGL NUC i7  </Name>
+		<ProductName> NUC11PAHi7 </ProductName>
```

Comment:
there should not be any space before and after product name.
<ProductName>NUC11PAHi7</ProductName>